src/overlays/actors/ovl_En_Kusa/z_en_kusa.h

```diff
@@ -7,12 +7,29 @@ struct EnKusa;
 
 typedef void (*EnKusaActionFunc)(struct EnKusa*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ ENKUSA_TYPE_BUSH,
+    /* 1 */ ENKUSA_TYPE_REGROW_GRASS,
+    /* 2 */ ENKUSA_TYPE_GRASS,
+    /* 3 */ ENKUSA_TYPE_GRASS_2
+} EnKusaType;
+
 typedef struct EnKusa {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ EnKusaActionFunc actionFunc;
-    /* 0x0148 */ char unk_148[0x54];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ EnKusaActionFunc actionFunc;
+    /* 0x148 */ ColliderCylinder collider;
+    /* 0x194 */ u8 timer;
+    /* 0x195 */ s8 objIndex;
+    /* 0x196 */ s8 kusaMtxIdx;
+    /* 0x197 */ u8 isCut;
+    /* 0x198 */ u8 unk198;
 } EnKusa; // size = 0x19C
 
 extern const ActorInit En_Kusa_InitVars;
 
+#define GET_KUSA_COLLECTIBLE_ID(x) (((x->actor.params >> 8) & 0x7F))
+#define GET_KUSA_RAND_COLLECTIBLE_ID(x) ((x->actor.params >> 8) & 0xF)
```

Comment:
```suggestion
#define GET_KUSA_COLLECTIBLE_ID(x) ((((x)->actor.params >> 8) & 0x7F))
#define GET_KUSA_RAND_COLLECTIBLE_ID(x) (((x)->actor.params >> 8) & 0xF)
```
This is safer, although these should not be passed anything weird. However, we usually make the macros to take `thisx` for consistency and to allow use fo them in the main 4 if necessary.